controllers/articles.controller.js

```diff
@@ -46,4 +51,22 @@ exports.postCommentsCon = (req, res, next) => {
         res.status(201).send({comment})
     })
     .catch((next))
+}
+
+exports.updateVotesCon = (req, res, next) => {
+    const votes = req.body
+    const endpoint = req.params.article_id
+    const promises = [updateVotesMod(votes, endpoint)]
+
+    if (endpoint){
```

Comment:
This is not required. If the endpoint has been visited, this function will fire and whatever is in the article_id position _will_ exist whether it is valid or not.